### DISCUSSION TITLE
Why can't I add a button?

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#3203] [Help]
I can't add a button, what's my mistake
![Снимок](https://user-images.githubusercontent.com/75489338/180379605-9e5b916c-6ef5-4196-af9d-8f137fc6ebef.JPG)
?

**Comment by danharrin** [comment]
You imported the wrong action class, read the error

**Reply by yablochnyi** [reply]
> You imported the wrong action class, read the error

I took it from the documentation
protected function getActions(): array
    {
        return [
            Actions\CreateAction::make(),
            Action::make('settings')->color('secondary')
        ];
    }

**Reply by danharrin** [reply]
Import `Filament\Pages\Actions\Action` instead of `Filament\Tables\Actions\Action`

**Reply by yablochnyi** [reply]
> Import `Filament\Pages\Actions\Action` instead of `Filament\Tables\Actions\Action`

Thanks)
